io/net/tcpdump.py

```diff
@@ -52,19 +57,47 @@ def setUp(self):
 
         # Install needed packages
         smm = SoftwareManager()
-        if not smm.check_installed("tcpdump") and not smm.install("tcpdump"):
-            self.cancel("Can not install tcpdump")
+        detected_distro = distro.detect()
+        pkgs = ['tcpdump']
```

Comment:
Please check and nmap package for RHEL distro. Few versions like RH7, does not have nmap installed by default.